### PR TITLE
Create a two org dev network demonstrator

### PR DESCRIPTION
**Pull request by paulwizviz**
- Produce a working demonstrator
- Update documentation

### CODE REVIEW

File: docs/networks/twoorgs-dev.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+STEP 7 - In `org1` cli terminal, run the command `./scripts/query.sh Paul` and you will see an indication stating Paul has 10 units.
+
+STEP 8 - Run the command `exit` to leave `org1` cli terminal.
+
+STEP 9 - Run the command `./fabricOps.sh cli org2` to give you access to `org2` cli terminal.
+
+STEP 10 - In `org2` cli terminal, run the command `./scripts/joinChan.sh` to join `org2` peer node to `mychannel`.
+
+STEP 11 - In `org2` cli terminal, run the command `./scripts/installChan.sh` in `org2` peer node.
```
Comment (dmcarrington) [inline docs/networks/twoorgs-dev.md]: I think this should be installCC.sh, not installChan.sh

**Review by dmcarrington — COMMENTED** [review]
Couple of commits are not signed, I think it is possible to do this retrospectively??
Noted what looks like one mistake in the documentation - see separate comment.
Aside from that, looks good, was able to follow the steps and get everything running.

**Comment by jagsk8** [comment]
As david mentioned - the README needs to be updated for Step 11: installCC.sh instead of installChan.sh.

Followed the step and was able to get the two org dev network up and running.

**Review by jagsk8 — APPROVED** [review]

**Review by jagsk8 — APPROVED** [review]

**Review by jagsk8 — DISMISSED** [review]

**Comment by paulwizviz** [comment]
@dmcarrington @jagsk8 Thanks for taking time off to review. I have made the changes requested now. Could you review it again, if you have time?

**Review by dmcarrington — APPROVED** [review]
Review passed following update.

**Pull request merged**